### ISSUE TITLE
Display returned values when invoking a Lambda

### ISSUE THREAD

**Issue by smacdav** [commandeer/open#91]
Handlers on Lambda functions can return a value either by directly returning something or by writing to an output stream. From the command line you can write this value to an output file, but in Commandeer, there is no indication that anything was returned.

The ask here is that there be a way to display values that are returned from an invoked Lambda.

**Comment by bwship** [comment]
@smacdav - is the ask on this to see the context body?

**Comment by smacdav** [comment]
Yes, let's go with that. That would definitely give me what I need.

**Comment by smacdav** [comment]
To clarify. A Lambda function could return just a String (or really any other object). In that case, I would like to be able to see the value returned.

Here's an example of a Java handler that returns a simple String:

```
public String handleRequest(String input) {
    return "Input received: " + input;
}
```

I would like to be able to see the value of that string.

Does that clarify?

**Comment by bwship** [comment]
@smacdav - it looks like this one was just a bug.  We have a fix for it coming in the next release.  Should be out by end of week,

**Comment by smacdav** [comment]
Thank you. I look forward to the release

**Comment by bwship** [comment]
Released in version 1.6
